src/model/nonce.ts

```diff
@@ -32,3 +37,42 @@ export const create = (nonce: Nonce) => (
     singleStringReply,
     falsyResponseToErrorAsync(new Error("Error saving the key"))
   );
+
+export const getNonceFromSignatureInput = (
```

Comment:
This isn't a model function. Is an utils to extract the value from the `request`.

Reply:
moved in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/f31eed4e20119c91d4e2b9977660f7823ed56d1a